ansible_rulebook/websocket.py

```diff
@@ -137,20 +157,26 @@ async def _handle_request_workload(
     activation_instance_id: str,
 ) -> StartupArgs:
     logger.info("workload websocket connected")
-    await websocket.send(
-        json.dumps(
-            dict(
-                type="Worker",
-                activation_id=activation_instance_id,
-                activation_instance_id=activation_instance_id,
+    try:
+        await websocket.send(
+            json.dumps(
+                dict(
+                    type="Worker",
+                    activation_id=activation_instance_id,
+                    activation_instance_id=activation_instance_id,
+                )
             )
         )
-    )
+    except Exception as e:
```

Comment:
@bzwei Can we be explicit about which exception we are catching, also we need a log line if we are catching Exception